src/routes/(console)/project-[region]-[project]/settings/domains/add-domain/+page.svelte

```diff
@@ -33,7 +33,7 @@
                 await goto(routeBase);
                 await invalidate(Dependencies.DOMAINS);
             } else {
-                await goto(`${routeBase}/add-domain/verify-${domainName}?rule=${rule.$id}`);
```

Comment:
We still need the rule, we use it in the verify page 
`const ruleId = page.url.searchParams.get('rule');`